ansible/roles/users/tasks/remove_users.yml

```diff
@@ -0,0 +1,41 @@
+---
+
+- name: Get running processes
+  shell: "ps aux | grep -v grep | grep -Ew \"{{ dev_users.absent|join('|') }}\" | awk '{print $2}'"
+  register: running_processes
+
+- name: Kill running processes
+  become: yes
+  shell: "kill {{ item }}"
+  with_items: "{{ running_processes.stdout_lines|default([]) }}"
+
+- wait_for:
```

Comment:
should make this timeout after 30 or 60s instead of 300

Reply:
Made thee timeout 15s (which is for each process)